project.clj

```diff
@@ -1,25 +1,22 @@
 (def VERSION "0.9.0-SNAPSHOT")
 
-(def VERSION-FORM `(do (require 'cider-nrepl.plugin)
```

Comment:
i deliberately just removed this as this deffo caused the problems i struggled with (see last comments on #143). If I understood project.clj right this magic was only used in `dev` profile... is that right? why do you need this in dev? what is the usual dev workflow which needs this trick?